source/simulator/parameters.cc

```diff
@@ -207,57 +207,63 @@ namespace aspect
                        "This parameter indicates whether the simulator should also use "
                        "heat conduction in determining the length of each time step.");
 
-    const std::string allowed_solver_schemes = "single Advection, single Stokes|iterated Advection and Stokes|"
-                                               "single Advection, iterated Stokes|no Advection, iterated Stokes|"
+    const std::string allowed_solver_schemes = "no Advection, no Stokes|"
                                                "no Advection, single Stokes|"
+                                               "no Advection, single Stokes first timestep only|"
+                                               "first timestep only, single Stokes|" // deprecated: use "no Advection, single Stokes first timestep only" instead
```

Comment:
Do you want to output a warning to the screen if someone uses this? The way it is, if a user does not look into the source code, they will never know that it's deprecated.